### DISCUSSION TITLE
Does gopro lab firmware consume more battery power?

### DISCUSSION THREAD

**Discussion by yzkzero** [gopro/labs#254] [Q&A] [answered]
I just flashed gopro9 lab firmware (1.72.70) yesterday. It is pretty cool.
But I found the battery remaining power dropped a little faster than before. So I'm wondering if the lab firmware consumes more power? Because it consistently recognize QR code in idle mode. I know the QR scanning is by default off during recording, so let's just talk about idle status.

**Comment by dnewman-gpsw** [comment, marked as answer]
Nothing detectable. QR decoding is only using low priority spare CPU cycles, nothing is turned extra to enable that work.  Labs enabled camera will run the same duration as stock cameras.

**Reply by yzkzero** [reply]
Thanks for the answer
